src/components/tagPropTypes.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** Name of a tag */
+    name: PropTypes.string.isRequired,
+
+    /** Flag that determines if a tag is active and able to be selected */
+    enabled: PropTypes.bool.isRequired,
```

Comment:
There's also an optional `GL Code` field that's a string

Reply:
Can it be described the same as the `GL Code` for categories?
```/** "General Ledger code" that corresponds to this category in an accounting system. Similar to an ID. */```

Reply:
Yep!